Azure/README.md

```diff
@@ -0,0 +1,39 @@
+# Clear Containers on Azure-VM
+
+This project allows to launch an Azure VM and install all Clear
+Container components and dependencies in order to run Clear Containers.
+
+# Requirements
+
+- Installation of azure-cli V 2.0 https://github.com/Azure/azure-cli.
+- Having previously logged into Azure.
+
+```bash
+$ az login
+```
+
+## Usage
+
+```bash
+$ bash cc-setup-azure.sh
+```
+
+## Options
+
+To see available options, run:
+
+```
+$ bash cc-setup-azure.sh -h
+```
+
+# Example
+
+```bash
+$ bash cc-setup-azure.sh -n MyTestVM -g MyTestGroup
+```
+
+# Login VM
+
+This script uses a default user name that can be overwritten.
+You can login to the VM using by SSH using the user name and
```

Comment:
You can log in to the VM by using the `SSH` command and providing the username and the assigned IP address.